.github/workflows/byob-gradle-trw.yml

```diff
@@ -0,0 +1,78 @@
+# Copyright 2023 SLSA Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+
+on:
+  workflow_call:
+    inputs:
+      name1:
+        description: "name 1"
+        required: false
+        type: string
+      name2:
+        description: "name 2"
+        required: false
+        type: string
+      rekor-log-public:
+        description: "private"
+        required: false
+        type: boolean
+        default: false
+      provenance-overwrite:
+        description: "overwrite provenance if already present"
+        required: false
+        type: boolean
+        default: false
+      release-tag: #test
+        description: "tag collision with dispatch event"
+        required: false
+        type: string
+        default: "v100"
+      artifact-list:
+        description: 'comma-separated list of paths to artifacts to be released. Use GRADLE_VERSION for the gradle version'
+        type: string
+        required: true
+      jdk-version:
```

Comment:
nit: use double quote for all description strings. (I think we have a linter that checks for this)